ide/csl.api/src/org/netbeans/modules/csl/editor/completion/MethodParamsTipPaintComponent.java

```diff
@@ -124,8 +124,8 @@ protected int getWidth(String s, Font font) {
     }
 
     protected int getHeight(String s, Font font) {
-        if (font == null) return fontMetrics.stringWidth(s);
-        return getFontMetrics(font).stringWidth(s);
+        if (font == null) return fontMetrics.getHeight();
+        return getFontMetrics(font).getHeight();
     }
```

Comment:
the method is never used. @neilcsmith-net @matthiasblaesing should I get rid of it? Package is not public. I am not completely sure if `getHeight()` is correct here since fonts are tricky - but it would be certainly more correct than width was.

Reply:
I stumbled over the fix, but the fix itself looks sane, so I ignored that.  I don't see how the method would be called, at least I have some hope that noone was "creative enough" to use reflection to invoke it. So I think this should be safe to remove. Maybe it was used sometime in the past and is a left over from a refactoring. The code in `setFont` could have been added later and there the font height is extracted.